nl.json

```diff
@@ -21,7 +20,7 @@
                 "December"
             ],
             "types" : {
-                "backpack" : "Rugzak",
+               "backpack" : "Rugzak",
```

Comment:
The spacing was correct before. Please don't remove the space before the translation key.